app/main.js

```diff
@@ -122,10 +148,17 @@ function shutdownDaemon(evenIfNotStartedByApp = false) {
       console.log('Killed lbrynet-daemon process');
     });
   } else if (evenIfNotStartedByApp) {
-    console.log('Killing lbrynet-daemon, even though app did not start it');
-    client.request('daemon_stop', []);
-    // TODO: If the daemon errors or times out when we make this request, find
-    // the process and force quit it.
+    console.log('Stopping lbrynet-daemon, even though app did not start it');
+    client.request('daemon_stop', [], (err, res) => {
+      if (err) {
+        // We could get an error because the daemon is already stopped (good)
```

Comment:
The problem is that we don't know which one is the appropriate one, unless we launched it, which we didn't necessarily.

We talked about finding the process listening on 5279 instead, but we don't want to risk killing someone else's process.

Hmm... what if we combined both: we keep this code to get a list of pids for all running lbrynet-daemons, then find the process listening on 5279, confirm it's in the list of daemon pids, then kill just that one? That should handle every scenario except the case where someone made their own app that happens be called "lbrynet-daemon" and also runs on 5279, in which case it's kind of their problem :P